iperf/README.md

```diff
@@ -0,0 +1,22 @@
+# iperf3 service on clearlinux
+Iperf is a widely used tool for network performance measurement and tuning.
+
+## Build
+```
+sudo docker build -t clr-iperf:latest .
+```
+
+## Run as Server:
+```
+sudo docker run -it --rm --name=iperf-srv -p 5201:5201 clr-iperf -s
```

Comment:
Same for the image name here.  Probably we want `clearlinux/iperf`?

Reply:
Yes, I think `clearlinux/iperf` is the expected name.